lib/fluent/plugin/in_forward.rb

```diff
@@ -57,11 +57,16 @@ def configure(conf)
     def start
       @loop = Coolio::Loop.new
 
-      @lsock = listen
+      socket_manager_path = ENV['SERVERENGINE_SOCKETMANAGER_PATH']
+      if Fluent.windows?
+        socket_manager_path = socket_manager_path.to_i
+      end
+      client = ServerEngine::SocketManager::Client.new(socket_manager_path)
```

Comment:
It's better to have a utility method for these several lines (it looks to appear very often), but I have a plan of a new plugin helper for tcp/udp/tls sockets. So it's my task following this pull-request.